web/src/pill_typeahead.ts

```diff
@@ -90,6 +90,7 @@ export function set_up_stream(
         items: 5,
         dropup: true,
         helpOnEmptyStrings: opts.help_on_empty_strings,
+        hideOnEmpty: true,
```

Comment:
Not sure how `hideOnEmpty: true`  and `helpOnEmptyStrings: true` will go together. I don't know if the PR behaves how alya suggested in 2nd point ([here](https://github.com/zulip/zulip/pull/24580#issuecomment-1458570050)). 


Even though her comment was on a different PR, both #28686 and #24580 are quite similar and I was told to look at it ([here](https://chat.zulip.org/#narrow/stream/101-design/topic/User.20invite.3B.20streams.20they.20should.20join/near/1724145)).